<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librepcb::BoardDesignRulesDialog</class>
 <widget class="QDialog" name="librepcb::BoardDesignRulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Board Design Rules</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Minimum</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ratio (% of diam.)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Maximum</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Stop Mask Max. Via Diam.:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="spbxStopMaskMaxViaDia">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Stop Mask Clearance:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spbxStopMaskClrMin">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="spbxStopMaskClrRatio">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="spbxStopMaskClrMax">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Cream Mask Clearance:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spbxCreamMaskClrMin">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="spbxCreamMaskClrRatio">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDoubleSpinBox" name="spbxCreamMaskClrMax">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Restring THT Pads:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="spbxRestringPadsMin">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="spbxRestringPadsRatio">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QDoubleSpinBox" name="spbxRestringPadsMax">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Restring Vias:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="spbxRestringViasMin">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="spbxRestringViasRatio">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QDoubleSpinBox" name="spbxRestringViasMax">
     <property name="suffix">
      <string>mm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999.999000000000024</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QPlainTextEdit" name="txtDescription"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="edtName"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>librepcb::BoardDesignRulesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>librepcb::BoardDesignRulesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
